### DISCUSSION TITLE
how to change to 0.0.0.0  when Getting Started with Docker

### DISCUSSION THREAD

**Discussion by dyh2024** [ItzCrazyKns/Vane#282] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/6c400275-51a5-452a-a798-2603fcd9a1d6)

How to change 'localhost' to '0.0.0.0' ?

**Comment by ItzCrazyKns** [comment, marked as answer]
Replace the start command in the package file with `npx next dev -H 0.0.0.0 -p 3000` and then rebuild the images (if you're using docker) by `docker compose up -d --build`.
